### DISCUSSION TITLE
cant see local ollama models in openweb ui

### DISCUSSION THREAD

**Discussion by jamieduk** [open-webui/open-webui#4632] [General]
# Bug Report

## Installation Method

[Describe the method you used to install the project, e.g., git clone, Docker, pip, etc.]

## Environment
64 bit linux

 ollama --version
ollama version is 0.3.3

i see the web ui but no models seem to be in the list!

- **Open WebUI Version:** [e.g., v0.3.11]
- **Ollama (if applicable):** [e.g., v0.2.0, v0.1.32-rc1]

- **Operating System:** [e.g., Windows 10, macOS Big Sur, Ubuntu 20.04]
- **Browser (if applicable):** [e.g., Chrome 100.0, Firefox 98.0]

**Confirmation:**

- [ ] I have read and followed all the instructions provided in the README.md.
- [ ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [ ] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

[Describe what you expected to happen.]

## Actual Behavior:

[Describe what actually happened.]

## Description

**Bug Summary:**
[Provide a brief but clear summary of the bug]

## Reproduction Details

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots/Screen Recordings (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by M0S4YSH3LL0** [comment]
i have the same issue..

Debian GNU/Linux 12 (bookworm) x86_64
ollama version is 0.3.11

**Comment by lmaddox** [comment]
Show configs please. I've got this working with multiple ollama urls on 192..../24 in a docker-compose stack. I, however, have not yet figured out the reverse proxy setup, so I probably can't help with that yet.

**Reply by M0S4YSH3LL0** [reply]
Already figured it out, had to enable host networking in docker and correct my docker run command, then it worked like a charm

**Reply by ssyrc** [reply]
@M0S4YSH3LL0, Could you please share your network configuration in sample and docker run command? I have the same issue in air-gapped network

**Reply by M0S4YSH3LL0** [reply]
the host networking is an option in Network configuration of Docker Desktop, just check it.
the run command I used:
`sudo docker run -d -p 3000:8080 -e OLLAMA_BASE_URL=http://192.xxx.xxx:11434 -e ENABLE_OLLAMA_API=true -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main`

replace `192.xxx.xxx` with the actual local ip of your ollama instance

**Reply by ssyrc** [reply]
You are my hero! :) I finally solved my problem with adding environments by -e OLLAMA_BASE_URL={my_IP}:11434
